client/src/App.js

```diff
@@ -2,12 +2,25 @@ import "./App.css";
 import "../node_modules/bootstrap/dist/css/bootstrap.min.css";
 import "../node_modules/bootstrap/dist/js/bootstrap.bundle";
 import Navbar from "./components/Navbar";
-function App() {
+import Home from "./components/Home";
+import Register from "./components/Register";
+import { Routes, Route } from "react-router-dom";
+import "../src/components/Register.js";
+import React from "react";
+import "../src/components/Edit.js";
+import Edit from "../src/components/Edit.js";
+
+const App = () => {
   return (
     <>
       <Navbar />
+      <Routes>
+        <Route exact path="/" element={<Home />} />
+        <Route exact path="/register" element={<Register />} />
```

Comment:
do not use hard text anywhere in code. define constant variables and use those variables here. 
variable name should be like registerRoute. editRoute, etc.